server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java

```diff
@@ -334,10 +334,46 @@ public void removeGroupMappingShouldDelete() throws Exception {
         virtualUserTable.removeMapping(source, Mapping.group(ADDRESS));
         virtualUserTable.removeMapping(source, Mapping.group(address2));
 
-        assertThat(virtualUserTable.getMappings(USER, domain))
+        assertThat(virtualUserTable.getUserDomainMappings(source))
             .isEqualTo(MappingsImpl.empty());
     }
 
+    @Test
+    public void addAliasMappingShouldStore() throws Exception {
+        Domain domain = Domain.LOCALHOST;
```

Comment:
We can inline that variable and use directly `Domain.LOCALHOST` instead...

Also, this smells copy-paste from previous test cases. This remark might apply there too IMO (In a separate commit?)